flake.nix

```diff
@@ -48,235 +48,245 @@
+              };
+          in
+          {
+            # Configure a formatter so that `nix fmt` can be used to format this file.
+            formatter = pkgs.nixpkgs-fmt;
+
+            packages = {
+              inherit holochain;
+              inherit lair-keystore;
+              inherit hc-launch;
+              inherit hc-scaffold;
+              inherit rust;
             };
-        in
-        {
-          # Configure a formatter so that `nix fmt` can be used to format this file.
-          formatter = pkgs.nixpkgs-fmt;
-
-          packages = {
-            inherit holochain;
-            inherit lair-keystore;
-            inherit hc-launch;
-            inherit hc-scaffold;
-            inherit rust;
-          };
 
-          # Define runnable applications for use with `nix run`.
-          # These can be used like `nix run "github:holochain/holonix#hc-scaffold" -- --version`.
-          # https://flake.parts/options/flake-parts.html?highlight=perSystem.apps#opt-perSystem.apps
-          apps = {
-            holochain.program = "${holochain}/bin/holochain";
-            hc.program = "${holochain}/bin/hc";
-            hc-run-local-services.program = "${holochain}/bin/hc-run-local-services";
-            hc-sandbox.program = "${holochain}/bin/hc-sandbox";
-            hcterm.program = "${holochain}/bin/hcterm";
-            lair-keystore.program = "${lair-keystore}/bin/lair-keystore";
-            hc-launch.program = "${hc-launch}/bin/hc-launch";
-            hc-scaffold.program = "${hc-scaffold}/bin/hc-scaffold";
-          };
+            # Define runnable applications for use with `nix run`.
+            # These can be used like `nix run "github:holochain/holonix#hc-scaffold" -- --version`.
+            # https://flake.parts/options/flake-parts.html?highlight=perSystem.apps#opt-perSystem.apps
+            apps = {
+              holochain.program = "${holochain}/bin/holochain";
+              hc.program = "${holochain}/bin/hc";
+              hc-run-local-services.program = "${holochain}/bin/hc-run-local-services";
+              hc-sandbox.program = "${holochain}/bin/hc-sandbox";
+              hcterm.program = "${holochain}/bin/hcterm";
+              lair-keystore.program = "${lair-keystore}/bin/lair-keystore";
+              hc-launch.program = "${hc-launch}/bin/hc-launch";
+              hc-scaffold.program = "${hc-scaffold}/bin/hc-scaffold";
+            };
 
-          devShells = {
-            default = pkgs.mkShell {
-              packages = [
-                holochain
-                lair-keystore
-                hc-launch
-                hc-scaffold
-                rust
-              ];
+            devShells = {
+              default = pkgs.mkShell {
+                packages = [
+                  holochain
+                  lair-keystore
+                  hc-launch
+                  hc-scaffold
+                  rust
+                ];
+              };
             };
           };
+      } // {
```

Comment:
From here is all that's actually changed in this file. It's just a spectacular confusion to the diff that the brace has moved above